auslib/db.py

```diff
@@ -2537,12 +2537,23 @@ class AUSDatabase(object):
     engine = None
     migrate_repo = path.join(path.dirname(__file__), "migrate")
 
-    def __init__(self, dburi=None, mysql_traditional_mode=False):
+    def __init__(self, dburi=None, mysql_traditional_mode=False, notify_tables=None):
         """Create a new AUSDatabase. Before this object is useful, dburi must be
            set, either through the constructor or setDburi()"""
         if dburi:
             self.setDburi(dburi, mysql_traditional_mode)
         self.log = logging.getLogger(self.__class__.__name__)
+        if notify_tables:
+            self.notify_tables = notify_tables
```

Comment:
How would this work ? Seems a bit like you're passing references to tables/attributes that don't exist yet, but perhaps I've forgotten how we instantiate all this.